WordPress/gutenberg
# NUX: Make tips appear inline

Implements part of #16315.

![localhost_8888_wp-admin_post-new php](https://user-images.githubusercontent.com/612155/61194952-6ce47800-a708-11e9-9265-c24eb0ec3f07.png)

Improve the usability of NUX tips by making them appear as inline notices (via a new `InlineTip` component) rather than floating popovers.

To test:

1. Create a new post
2. Click _More tools & options_
3. Select _Options_
4. Check _Enable Tips_
